specification/schemas/returns/ReturnSimpleResponse.json

```diff
@@ -47,6 +47,56 @@
               "example": "$API_HOST/returns/v1/returns/8ddfb83f-d600-49c7-b10e-66dd4dc7686a"
             }
           }
+        },
+        "meta": {
+          "readOnly": true,
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "shipment": {
+              "type": "object",
+              "additionalProperties": false,
+              "required": [
+                "tracking_code",
+                "status"
+              ],
+              "properties": {
+                "tracking_code": {
+                  "type": "string",
+                  "description": "Tracking code of the shipment",
+                  "example": "3SDFGDFGDFGDFG"
+                },
+                "status": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "code": {
+                      "type": "string",
+                      "enum": [
```

Comment:
Instead of defining the entire enum again, let's make a `StatusCode` schema and use that in both places:
```
"code": {
  "$ref": "#/components/schemas/StatusCode"
}
```

Reply:
That wouldn't be correct because the StatusCode schema includes status codes for other resources, other than shipments.

The resulting duplication is very small and imo manageable.